nodebuilder/state/cmds/state.go

```diff
@@ -414,11 +369,20 @@ var queryRedelegationCmd = &cobra.Command{
 			return fmt.Errorf("error parsing a dst address:%v", err)
 		}
 
-		response, err := client.State.QueryRedelegations(
+		response, err := rpcClient.State.QueryRedelegations(
 			cmd.Context(),
 			srcAddr.Address.(state.ValAddress),
 			dstAddr.Address.(state.ValAddress),
 		)
-		return printOutput(response, err, nil)
+		return util.PrintOutput(response, err, nil)
 	},
 }
+
+func parseAddressFromString(addrStr string) (state.Address, error) {
```

Comment:
i've seen and used this in a few places as i've been looking around, again, i think there might be a compelling argument to create an `address.FromString(addr string) (state.Address error)` in the address package as part of this PR

Reply:
Yes, this makes sense. We can have a helper function somewhere in state module